src/components/EntityPage/Participant/Utils/OtherDataTypesSummaryTable.js

```diff
@@ -61,7 +60,7 @@ const OtherDataTypesSummaryTable = ({ files, participantID }) => {
   return arr.length === 0 ? (
     ''
   ) : (
-    <EntityContentSection title="Other Data Types" size={'small'}>
+    <EntityContentSection title={hasSequencingData ? "Other Data Types" :  "Data Types"} size={'small'}>
```

Comment:
Another way to do this could be:
`{hasSequencingData ? 'Other ' :  ''}Data Types`. :)